package.json

```diff
@@ -1,56 +1,66 @@
 {
-  "name": "cms-bb2",
-  "version": "1.0.0",
-  "description": "An sdk used for interacting with the CMS Blue Button 2.0 API",
-  "main": "dist/my-lib.cjs.js",
-  "module": "dist/my-lib.esm.js",
-  "browser": "dist/my-lib.umd.js",
-  "files": [
-    "dist"
-  ],
-  "scripts": {
-    "dev": "rollup -c -w",
-    "build": "rollup -c",
-    "build:types": "tsc -p ./tsconfig.json --outDir build --declaration true && api-extractor run",
-    "docs": "typedoc src --out docs",
-    "test": "jest --coverage",
-    "prepare": "husky install"
-  },
-  "lint-staged": {
-    "**/*": "prettier --write --ignore-unknown"
-  },
-  "jest": {
-    "preset": "ts-jest"
-  },
-  "keywords": [
-    "bb2",
-    "cms",
-    "medicare",
-    "claims"
-  ],
-  "author": "",
-  "license": "ISC",
-  "devDependencies": {
-    "@rollup/plugin-commonjs": "^21.0.1",
-    "@rollup/plugin-node-resolve": "^13.1.3",
-    "@rollup/plugin-typescript": "^8.3.0",
-    "@types/jest": "^27.4.0",
-    "@typescript-eslint/eslint-plugin": "^5.10.0",
-    "@typescript-eslint/parser": "^5.10.0",
-    "eslint": "^8.7.0",
-    "eslint-config-prettier": "^8.3.0",
-    "husky": "^7.0.4",
-    "jest": "^27.4.7",
-    "lint-staged": "^12.2.2",
-    "prettier": "2.5.1",
-    "rollup": "^2.64.0",
-    "ts-jest": "^27.1.3",
-    "ts-node": "^10.4.0",
-    "tslib": "^2.3.1",
-    "typescript": "^4.5.4"
-  },
-  "dependencies": {
-    "@microsoft/api-extractor": "^7.19.4",
-    "typedoc": "^0.22.11"
-  }
-}
+    "name": "cms-bb2",
```

Comment:
Why did you change all the spacing here? I'm guessing that happened because you removed the linting check from the precommit file?

Reply:
now linting re-surrected.